Realm.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
   s.osx.vendored_library    = 'core/librealm.a'
 
   s.subspec 'Headers' do |s|
-    s.source_files          = 'include/**/*.h'
+    s.source_files          = 'include/**/*.{h,hpp}'
```

Comment:
If they're private, that means they'll ship inside the build framework (as will all the headers in core they depend on), I believe.